pkg/action/install_test.go

```diff
@@ -214,6 +196,10 @@ var _ = Describe("Install action tests", func() {
 			installer = action.NewInstallAction(config, spec)
 		})
 		AfterEach(func() {
+			if CurrentSpecReport().Failed() {
+				GinkgoWriter.Printf(memLog.String())
+			}
+			Expect(os.RemoveAll("/tmp/test.img")).ToNot(HaveOccurred())
```

Comment:
Maybe worth making it a var and setting the value at the top level `BeforeEach`. It will make it easier to change the location of that file if we need to in the future (e.g. if tests start conflicting with each other because of the non-random name).